routers/web/repo/setting/webhook.go

```diff
@@ -425,15 +425,19 @@ func TelegramHooksEditPost(ctx *context.Context) {
 func telegramHookParams(ctx *context.Context) webhookParams {
 	form := web.GetForm(ctx).(*forms.NewTelegramHookForm)
 
+	if form.CustomHost == "" {
+		form.CustomHost = "https://api.telegram.org"
```

Comment:
convert the default server host into a const. and add it to `ctx.Data`, then we can use it in template.